integration/resource_lacework_alert_channel_webhook_test.go

```diff
@@ -0,0 +1,32 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestWebhookAlertChannelCreate applies integration terraform '../examples/resource_lacework_alert_channel_webhook'
+// Uses the go-sdk to verify the created integration
+// Applies an update with new channel name and Terraform destroy
+func TestWebhookAlertChannelCreate(t *testing.T) {
+	terraformOptions := terraform.WithDefaultRetryableErrors(t, &terraform.Options{
+		TerraformDir: "../examples/resource_lacework_alert_channel_webhook",
+		Vars: map[string]interface{}{
+			"channel_name": "Webhook Alert Channel Example",
+		},
+	})
+	defer terraform.Destroy(t, terraformOptions)
+
+	// Create new Webhook Alert Channel
+	create := terraform.InitAndApplyAndIdempotent(t, terraformOptions)
+	assert.Equal(t, "Webhook Alert Channel Example", GetIntegrationName(create))
+
+	// Update Webhook Alert Channel
+	terraformOptions.Vars = map[string]interface{}{
+		"channel_name": "Webhook Alert Channel Updated"}
```

Comment:
Sounds good! Though I still like having the integration retrieved with the CLI as well.

Reply:
Yeah, we agreed on both, 1) Terraform outputs since we are testing our terraform provider and 2) Retrieve integrations' data from the server via the Go client inside the SDK, to verify that the modifications were made correctly.

You can see how @vatasha also use [this function to get the props](https://github.com/lacework/terraform-provider-lacework/blob/main/integration/resource_lacework_alert_channel_datadog_test.go#L40) field from the server:
```go
 	updateProps := GetAlertChannelProps(update)
```